### PR TITLE
tests: reset the fuzzer scenario and export/import for regressions

### LINKED ISSUE #3206: tests: fuzzer: find a way to add stdcm tests as regresion tests
When the fuzzer detects an error, a json file is generated so that it can be included in the regression test suite.

But stdcm tests depend on the current state of the timetable, they can't easily be reproduced in a vacuum. We could look for a way to generate a timetable with a predefined set of trains, to run reproducible stdcm tests.

### PR DESCRIPTION
**Pull request by shenriotpro**
This is the first step to reproduce STDCM errors: we need a reasonable timetable.

edit: I went ahead and also made the export/import for regressions, see https://github.com/osrd-project/osrd/issues/3206

### CODE REVIEW

File: tests/tests/test_regressions.py [hunk 1]
```diff
@@ -50,32 +76,30 @@ def _reproduce_test(path_to_json: Path, scenario: Scenario, rolling_stock_id: in
     fuzzer_output = json.loads(path_to_json.read_bytes())
     _load_infra(EDITOAST_URL, scenario.infra)
 
+    assert fuzzer_output["infra_name"] in ["small_infra", "Small Infra"]
+    timetable = scenario.timetable
+
     if fuzzer_output["error_type"] == "STDCM":
+        _apply_prelude(fuzzer_output.get("prelude", []), EDITOAST_URL, scenario.infra, timetable, rolling_stock_id)
         _stdcm_with_payload(EDITOAST_URL, fuzzer_output["stdcm_payload"])
```
Comment (eckter) [inline tests/tests/test_regressions.py]: The stdcm payload needs to be modified a little now, it needs to have the right infra / rollingstock / timetable ids

**Review by eckter — COMMENTED** [review]
Nice job, very clean code

I can't quite reproduce STDCM bugs, tests that fail in the fuzzer can pass in the regression test suite. I've identified one issue (mismatch in the infra/schedule ids).

File: tests/tests/test_regressions.py (hunk 1 above)
Comment (shenriotpro) [reply tests/tests/test_regressions.py]: Thanks. Apparently it's not enough to reproduce 500 errors from the fuzzer, inconsistent rolling stock might be an issue, I'll investigate a bit more.

Comment (shenriotpro) [reply tests/tests/test_regressions.py]: Indeed it works best when consistently using fast_rolling_stock, let me see what I can do.

File: tests/fuzzer/fuzzer.py [hunk 2]
```diff
@@ -181,16 +206,11 @@ def test_stdcm(
     stdcm_payload = make_stdcm_payload(scenario, path, rolling_stock)
     r = post_with_timeout(editoast_url + "stdcm/", json=stdcm_payload)
     if r.status_code // 100 != 2:
-        if r.status_code // 100 == 4 and "no_path_found" in r.content.decode("utf-8"):
+        content = r.content.decode("utf-8")
+        if r.status_code // 100 == 4 and ("no_path_found" in content or "No path could be found" in content):
```
Comment (eckter) [inline tests/fuzzer/fuzzer.py]:
```suggestion
        if r.status_code // 100 == 4 and "No path could be found" in content:
```
The detailed error message should remain there even when the error code will be fixed

**Review by eckter — APPROVED** [review]

**Pull request merged**